### DISCUSSION TITLE
What do you do when you have multiple KSamplers in a workflow and you want to run any one of them?

### DISCUSSION THREAD

**Discussion by nefudev** [Comfy-Org/ComfyUI#1635] [Q&A]
Hello.

I am implementing many functions in one Workflow. I don't have multiple Workflows for each function because file management is a pain.
But there is a problem, for example, if I have a flow with multiple KSamplers, even if I want to run only one, all of them will be executed.

Therefore, the KSampler that we do not want to run is temporarily disconnected from the Latent connection to avoid the problem.
However, this is not smart. Is there a better way?

**Comment by Mistralys** [comment]
In vanilla ComfyUI, you cannot select single KSamplers to run again. That feature typically comes from a widely used custom node pack called "rgthree's ComfyUI Custom Nodes". This adds a lot of quality-of-life features, including the ability to selectively queue nodes or branches of your workflow.

Typically, you will want to install this extension using the [ComfyUI Manager](https://github.com/Comfy-Org/ComfyUI-Manager).
